### DISCUSSION TITLE
Pants giving diet recently stopped eating to fill new stomach space

### DISCUSSION THREAD

**Discussion by PharmerGord** [loathers/garbage-collector#1993] [Q&A]
[tyrone_jones_20240807.txt](https://github.com/user-attachments/files/16532855/tyrone_jones_20240807.txt)
I am struggling with each time pantsgiving gains an additional stomach space the script gets caught in a loop. I usually try to check periodically, then stop all, use CONSUME to eat again and then reenter the script. 

Any ideas on how I can try to minimize this without going to closeting my pantsgiving?

**Comment by spaghetti-squash** [comment]
This feels like it's likely worth a bug report. The completely silent failure suggests that we're generating a completely empty diet entry. Is your spleen or liver overfull? It doesn't look like you're overdrunk for this.
